File: Python/ReverseNumber.py
def reverse_number(number: int) -> int:
    reversed_number = 0
    while number != 0:
        digit = number % 10
        reversed_number = reversed_number * 10 + digit
        number //= 10
    return reversed_number

print(reverse_number(1234))
print(reverse_number(6543))

File: CPP/ReverseNumber.cpp
#include <iostream>
using namespace std;

int reverseNumber(int number) {
    int reversed = 0;
    while (number != 0) {
        int digit = number % 10;
        reversed = reversed * 10 + digit;
        number /= 10;
    }
    return reversed;
}

int main() {
    cout << reverseNumber(1234) << endl;
    cout << reverseNumber(6543) << endl;
}